test/e2e-common.sh

```diff
@@ -43,18 +43,20 @@ function istio_yaml() {
   local istio_mesh=$2
   local suffix=""
   if [[ $istio_mesh -eq 0 ]]; then
-    suffix="-lean"
+    suffix="ci-no-mesh"
+  else
+    suffix="ci-mesh"
   fi
-  echo "third_party/istio-${istio_version}/istio${suffix}.yaml"
+  echo "third_party/istio-${istio_version}/istio-${suffix}.yaml"
 }
 
 # Install Istio.
 function install_istio() {
-  local base_url="https://raw.githubusercontent.com/knative/serving/v${LATEST_SERVING_RELEASE_VERSION}"
+  local base_url="https://raw.githubusercontent.com/knative/serving/${LATEST_SERVING_RELEASE_VERSION}"
   # Decide the Istio configuration to install.
   if [[ -z "$ISTIO_VERSION" ]]; then
```

Comment:
can this ever be false? isn't it always set and can't be overwritten?

Reply:
Will remove it.